src/tables.jl

```diff
@@ -91,19 +91,23 @@ function read(io_or_path; validate::Bool=true)
 end
 
 """
-    Legolas.write(io_or_path, table, schema::Schema; validate::Bool=true, kwargs...)
+    Legolas.write(io_or_path, table, row_or_schema; validate::Bool=true, kwargs...)
```

Comment:
Let’s not do this and instead keep the input domain simple for callers; this function really only needs the schema to do its job, not a row, so we should only have it accept a schema

Rather the “right” thing to do is to solve pain points with extracting/constructing schemas themselves (which the rest of this PR is adding mechanisms for)